package.json

```diff
@@ -4,8 +4,9 @@
   "description": "ESLint rules for Primer React",
   "main": "src/index.js",
   "scripts": {
-    "test": "jest",
-    "release": "changeset publish"
+    "lint": "eslint .",
```

Comment:
Do we want to use eslint for the markdown files as well or do we want to filter them out and use specific linter we have in primer/react?

`"lint:md": "markdownlint-cli2 \"**/*.{md,mdx}\" \"!.github\" \"!.changeset\" \"!node_modules\" \"!CHANGELOG.md\"",`

Reply:
Ooo great idea, let me see how hard it would be and if it's easy to bring in I'll do it in this PR otherwise I'll open up another one 👍

Reply:
Added! ✨

Reply:
yay! ✨